src/components/color_picker/_color_picker.scss

```diff
@@ -16,17 +16,3 @@
   // sass-lint:disable no-important
   padding-bottom: 0 !important;
 }
-
-.euiColorPicker__popoverPanel[class*='bottom']:not(.euiColorPicker__popoverPanel--customButton) {
```

Comment:
I see you're removing this from the color picker sass, but didn't see it actually changing in the color picker js?

Reply:
Moved to `popover.scss` with a new class set in that components

Reply:
Ahh, I'm tracking now